data.md

```diff
@@ -155,6 +158,21 @@ Functions `#signed` and `#unsigned` allow for easier operation on twos-complemen
     rule #unsigned(ITYPE, N) => N                  requires 0 <=Int N
 ```
 
+These ranges help check whether a K integer is in the correct ranges for Wasm integers.
+
+```k
+    syntax Int ::= #minSigned  ( IValType ) [function]
+ // --------------------------------------------------
+    rule #minSigned(ITYPE) => 0 -Int #pow1(ITYPE)
```

Comment:
Is this the cleanest way in K to negate an integer?

Reply:
Yes

Reply:
It will actually execute in constant time too because k uses gmplib for integers which usus sign and magnitude.